src/Relude/Lifted/IORef.hs

```diff
@@ -8,6 +8,9 @@ License:    MIT
 Maintainer: Kowainik <xrom.xkov@gmail.com>
 
 Lifted reexports from 'Data.IORef' module.
+
+Lifted meaning that you can also use them inside various
```

Comment:
Let's keep this comment consistent with other identical comments :slightly_smiling_face: